osd/uwm_misconfiguration.json

```diff
@@ -4,7 +4,7 @@
   "log_type": "cluster-configuration",
   "_tags": ["t_config"],
   "summary": "Action required: review user-workload-monitoring configuration",
-  "description": "Your cluster's user-workload-monitoring is misconfigured. ${REASON}. Please review your configuration according to https://docs.openshift.com/rosa/monitoring/configuring-the-monitoring-stack.html .",
+  "description": "Your cluster's user workload monitoring is misconfigured. ${REASON}. Please review your configuration alongside the product documentation.",
   "doc_references": ["https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/monitoring/configuring-the-monitoring-stack"],
```

Comment:
I see that we have removed the doc link from the service log `description`, since it exists in the `doc_reference` link. (this is one example, but there are many others)
Before we do this, we should make sure that the SLs sent by SRE will contain documentation from the `doc_references` automatically included. I know there were some conversation around this, @laubai  can you confirm if customer will get doc links from `doc_references` when SREs send service logs?

Trying to avoid losing this information when SLs are sent when we communicate with customers. :slightly_smiling_face:

Reply:
I believe so but I'm not certain - let me revert the cases where I've done this until we're sure.

Reply:
@karthikperu7 Reverted all the cases I had changed initially, while keeping the doc_references. Also fixed a few things that will likely trigger 404 checking (e.g. full stop at the end of a URL)